src/app/features/profile/profile-detail/profile-detail.component.ts

```diff
@@ -14,8 +16,15 @@ import { Observable } from 'rxjs';
 export class ProfileDetailComponent implements OnInit {
 
     public profile$: Observable<IProfile>;
+    public isLoaded$ = new Subject<boolean>();
+
+    constructor (private store: Store<AppState>, public actions$: Actions) {
+
+        actions$.pipe(
+            ofType(getProfileSuccess, getProfileFail),
+            take(1)
+        ).subscribe(() => this.isLoaded$.next(true));
```

Comment:
you need to unsubscribe even from `take(1)` because it may doesn't emit even 1 value